docs/development_environment.mdx

```diff
@@ -59,6 +59,24 @@ If the Dev Container was set up correctly - it supports debugging by default, ou
 
 It is also possible to debug a remote Home Assistance instance (e.g., production instance) by following the procedure described [here](https://www.home-assistant.io/integrations/debugpy/).
 
+### Use devcontainer from WSL
```

Comment:
we need to ensure, that the user is using a WSL2 (_hyper-v based_) instead of the legacy WSL1 (_dirty linux kernel to windows translation_)

Reply:
I've added a link to some instructions that help with this.